R/ATTAINSCrosswalks.R

```diff
@@ -1239,24 +1292,24 @@ TADA_CreateUseParamRef <- function(.data, org_id = NULL, paramRef = NULL,
   # Users will need to have ran TADA_GetATTAINS() for this option to be allowed.
   # Selection of org_id will filter the drop down lists in future steps of creating the reference tables.
   if (is.null(org_id)) {
-    stop("TADA.CreateParamUseRef: No organization identifier(s) provided.")
+    stop("TADA.CreateUseParamRef: No organization identifier(s) provided.")
   }
 
   org_id <- as.list(org_id)
 
   # Checks if org_id are valid names found in ATTAINS - with the exception of "EPA304a" as that is not an ATTAINS org_id.
   if (sum(!org_id[org_id != "EPA304a"] %in% ATTAINS_param_all$organization_identifier) > 0) {
     warning(paste0(
-      "TADA_CreateParamRef: ",
+      "TADA_CreateuseParamRef: ",
       "One or more organization identifiers entered by user is not found in ATTAINS. "
     ))
   }
 
   # Checks if org_id are found in the user supplied paramRef argument.
   if (sum(!org_id[org_id != "EPA304a"] %in% paramRef$organization_identifier) > 0) {
     warning(paste0(
-      "TADA_CreateParamRef: ",
-      "One or more organization identifiers entered by user is not found in your paramRef argument input.",
+      "TADA_CreateUseParamRef: ",
```

Comment:
I wonder if this should be a "stop" instead of a "warning"? Because users cannot generate the ref for orgs missing from paramRef